CMakeLists.txt

```diff
@@ -18,19 +18,37 @@ find_package(Amlogic)
 find_package(Rockchip)
 
 find_package(PkgConfig REQUIRED)
+
+option(DISABLE_SDL "Don't compile SDL support" OFF)
+option(DISABLE_SOFTWARE "Disable software rendering support" OFF)
```

Comment:
Let's use `FFMPEG` rather than `SOFTWARE` here and "FFmpeg decoding support" rather than "software rendering support"

We do support VAAPI hardware acceleration, so calling it "software" isn't really accurate.